ee/apps.py

```diff
@@ -1,6 +1,19 @@
 from django.apps import AppConfig
 
+from ee.settings import VERSION_REQUIREMENTS_EE
+
 
 class EnterpriseConfig(AppConfig):
     name = "ee"
     verbose_name = "Enterprise"
+
+    def ready():
```

Comment:
`check_prerequisites()`?  Then we can technically add multiple checks (like `check_service_version_requirement`...)

Reply:
`ready` is a Django hook that will be called when the app is set up and ready

Reply:
anyway, removed from here